docs/running-in-production-dockerized.md

```diff
@@ -0,0 +1,100 @@
+# Running in Production Dockerized

```

Comment:
There's a lot of duplication between this and the existing "Running in production" page. I'd recommend leaving most of this information in the existing page and moving this to a separate page that's focused on specifically what's necessary for Docker Compose.